initial.c

```diff
@@ -0,0 +1,13 @@
+#include <initials.c> /*Sheqi Zhang 2016/06/29*/
+
+main()
+{
+	printf("Hello Z-world!/n")
+	printf("******/n")
+	printf("    */n")
+	printf("   */n")
+	printf("  */n")
+	printf(" */n")
+	printf("*/n")
+	printf("******/n")
+}
```

Comment:
Hi Sheqi - your code neither compiled nor ran. You should see me or Ms. Hsu about your issues with this assignment.